### PR TITLE
Internal: Kit defaults - Review fixes [ED-8532]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: modules/kit-elements-defaults/assets/js/editor/commands/create.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { updateElementDefaults } from '../api';
 import { isPopulatedObject } from '../utils';
```
Comment (StyleShit) [inline modules/kit-elements-defaults/assets/js/editor/commands/create.js:2]: remove unsued import

File: modules/kit-elements-defaults/assets/js/editor/extract-container-settings.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			return acc;
+		}
+
+		return {
+			...acc,
+			[ type ]: specialSettings,
+		};
+	}, {} );
+}
+
```
Comment (StyleShit) [inline modules/kit-elements-defaults/assets/js/editor/extract-container-settings.js:46]: you have 2 empty lines at the end of the file

File: tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php [hunk 3]
```diff
@@ -183,6 +183,27 @@ public function test_update_item__sanitizes_globals_and_dynamics() {
...
 	}
 
+	public function test_update_item__sanitizes_invalid_html_elements() {
+		// Arrange.
+		$this->act_as_admin();
+
+		// Act.
+		$response = $this->send_request( 'PUT', '/kit-elements-defaults/button', [
+			'settings' => [
+				'text' => '<script>alert("error")</script> Some text after',
```
Comment (StyleShit) [inline tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php:193]: maybe also add some before text?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/kit-elements-defaults/assets/js/editor/commands/create.js (hunk 1 above)
Comment (Nevoss) [reply modules/kit-elements-defaults/assets/js/editor/commands/create.js:2]: Done

File: modules/kit-elements-defaults/assets/js/editor/extract-container-settings.js (hunk 2 above)
Comment (Nevoss) [reply modules/kit-elements-defaults/assets/js/editor/extract-container-settings.js:46]: Done

File: tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php (hunk 3 above)
Comment (Nevoss) [reply tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php:193]: ?

Comment (StyleShit) [reply tests/phpunit/elementor/modules/kit-elements-defaults/data/test-controller.php:193]: You add text after the script. maybe also add some text before the script

**Review by StyleShit — APPROVED** [review]

**Pull request merged**